quaint/Cargo.toml

```diff
@@ -115,16 +115,10 @@ optional = true
 branch = "vendored-openssl"
 
 [dependencies.rusqlite]
-version = "0.25"
+version = "0.29"
 features = ["chrono", "bundled", "column_decltype"]
 optional = true
 
-[dependencies.libsqlite3-sys]
-version = "0.22"
-default-features = false
-features = ["bundled"]
-optional = true
```

Comment:
This is now included in newer versions of rusqlite?

Reply:
It already was! I'm not sure why we were doing it like this. If you check `error.rs`, we're just using the re-exported ffi from rusqlite now